actions/check-thirdparty-notice/action.yml

```diff
@@ -0,0 +1,76 @@
+# SPDX-FileCopyrightText: Copyright (C) 2025 Opal Health Informatics Group <https://www.opalmedapps.com>
+#
+# SPDX-License-Identifier: MIT
+
+# Ensures that declared project dependencies and the third-party notice in THIRDPARTY.md are in sync.
+# I.e., each dependency has to have a corresponding heading in the notice.
+# Warns about dependencies missing as well as dependencies that are declared in addition.
+#
+# Note: Currently supports npm, composer and pip.
+
+name: Check Thirdparty Notice
+description: Ensures that declared project dependencies and the third-party notice in THIRDPARTY.md are in sync.
+
+inputs:
+  package-file:
+    description: The filename of an npm package.json file (can be relative to the current working directory)
+    default: ""
+  composer-file:
+    description: The filename of a composer.json file (can be relative to the current working directory)
+    default: ""
+  pip-requirements-files:
+    description: |
+      The filename(s) of a requirements file (can be relative to the current working directory).
+      If specifying more than one file, separate with spaces.
+    default: ""
+  additional-dependencies:
+    description: |
+      Additional dependencies that are allowed to be listed in the third-party notice.
+      If specifying more than one package, separate with spaces.
+    default: ""
+
+runs:
+  using: "composite"
+  steps:
+    # Setup
+    # Copy check_thirdparty_notice.py from the action path to the current directory, so it can see all relevant files
+    - name: Copy check_thirdparty_notice.py to the current directory
+      run: cp "$GITHUB_ACTION_PATH/check_thirdparty_notice.py" .
+      shell: bash
+      env:
+        GITHUB_ACTION_PATH: ${{ github.action_path }}
+    - name: Install uv
+      uses: astral-sh/setup-uv@v6.1.0
+    - name: Make sure a dependencies file path was provided
+      run: |
+        if [[ -z "$PACKAGE_FILE" ]] \
+        && [[ -z "$COMPOSER_FILE" ]] \
+        && [[ -z "$PIP_REQUIREMENTS_FILES" ]]; then
+          echo "At least one of the inputs package-file, composer-file or pip-requirements-files must be provided";
+          exit 1;
+        fi
+      env:
+        PACKAGE_FILE: ${{ inputs.package-file }}
+        COMPOSER_FILE: ${{ inputs.composer-file }}
+        PIP_REQUIREMENTS_FILES: ${{ inputs.pip-requirements-files }}
+      shell: bash
+
+    - name: Compose the arguments to the script
+      run: | # zizmor: ignore[github-env]
```

Comment:
Is this coming from writing to `$GITHUB_ENV`? Have you tried using output instead?

Reply:
That worked, thanks for the suggestion.